### DISCUSSION TITLE
OptiX

### DISCUSSION THREAD

**Discussion by toaster-robotics** [NVIDIA/warp#292] [Q&A]
Hello, I'm using warp for a ray tracing simulation code. I read that OptiX enabled GPUs have hardware specifically to accelerate ray tracing calculations. I've poked around here and the docs but don't see any mention of OptiX. Is this just abstracted away? Or is it something not currently enabled by warp? Thanks!

**Comment by shi-eric** [comment]
Hi @toaster-robotics, we don't currently make use OptiX in Warp.

**Reply by toaster-robotics** [reply]
😭
